docker-compose.yml

```diff
@@ -32,3 +35,29 @@ services:
       - 5555:5432
     environment: *credentials
     command: ["postgres", "-c", "log_statement=all"]
+  
+  prest:
+    image: prest/prest:latest
+    ports:
+      - 8081:8081
+    depends_on:
+      postgres:
+        condition: service_healthy
+    
+    environment:
+       - PREST_DEBUG=true
+       - PREST_AUTH_ENABLED=false
+       - PREST_PG_HOST=postgres
+       - PREST_PG_USER=minhareceita
+       - PREST_PG_PASS=minhareceita
+       - PREST_PG_DATABASE=minhareceita
+       - PREST_PG_PORT=5432
+       - PREST_SSL_MODE=disable
+       - PREST_CACHE_ENABLED=false
+       - PREST_CACHE=false
+       - PREST_HTTP_PORT=8081
+       - PREST_QUERIES_LOCATION=/prest/data/queries/
+       - PREST_CONF=/prest/data/config/prest.toml
+    
+    volumes:
+      - ./prest/:/prest/data/
```

Comment:
Semanticamente o pREST vai substituir o _pacote_ `api/` — pq não usarmos o mesmo diretório?

Reply:
Não sei te responder se é possível incluir ele direto no código assim como um git submodule. Precisamos conversar e entender melhor se isso faz sentido. Se for apenas semântica de arquivos, tudo bem, podemos mover.

Reply:
Não é nem incluir no módulo, nem usar Git submódulo.

É manter a semântica da organização de arquivos atual com essas duas mudanças:

### 1

```suggestion
      - ./api/:/prest/data/
```

### 2

E `git mv prest/config/prest.toml api/config/prest.toml`.